.github/workflows/scenario-test.yaml

```
@@ -6,6 +6,10 @@ on:
   push:
     branches:
       - main
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

```suggestion
  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
```

https://docs.github.com/en/actions/reference/workflows-and-actions/contexts#github-context

> 	The name of the event that triggered the workflow run.

This creates a unique group for every combination of:
1. The workflow name
2. The event type (push, schedule, workflow_dispatch)
3. The branch/ref (e.g., refs/heads/main)

---

This way we can avoid cancelling sibling-workflows unintentionally.

---

@xmfcx Great suggestion! I added `github.event_name` into the concurrency key. 1b9340184b75abce005fac65f460fc2117d75364